skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -4,6 +4,8 @@
 #
 #
 #
+# Distributed under the terms of the none license.
+# See LICENSE.txt for more info.
```

Comment:
We don't want these two lines to appear.  Unfortunately POGO outputs them, but they need to be manually removed.  There is no "none" license, nor a `LICENSE.txt` file.  Our license in is a file called `COPYING`.